tests/lib/c_lib/src/main.c

```diff
@@ -1236,3 +1236,18 @@ ZTEST(test_c_lib, test_exit)
 	k_thread_abort(tid);
 	zassert_equal(a, 0, "exit failed");
 }
+
+ZTEST(test_c_lib, test_clock)
+{
+	clock_t then;
+	clock_t now;
+
+	then = clock();
+#ifdef CLOCKS_PER_SEC
```

Comment:
Shouldn't this just use 1000000 instead of a couple of constants that aren't measured in µs?